views/js/runner/plugins/tools/preventDropToInput.js

```diff
@@ -55,16 +55,37 @@ define([
                 .after('renderitem', function () {
                     var config = _.defaults((self.getConfig() || {}), _defaults);
                     var $items = testRunner.getAreaBroker().getContentArea().find(config.selector);
+                    var dropped, disabled;
+
+                    $('img')
+                        .off('.preventdropimg')
+                        .on(namespaceHelper.namespaceAll('dragstart', 'preventdropimg'), function (event) {
+                            dropped = event.target;
+                        })
+                        .on(namespaceHelper.namespaceAll('dragend', 'preventdropimg'), function (event) {
+                            dropped = null;
+                            disabled.prop('disabled', false);
+                        });
 
                     $items
-                        .off('.preventdrop')
-                        .on(namespaceHelper.namespaceAll('drop dragover', 'preventdrop'), function (event) {
-                            event.preventDefault();
+                        .off('.preventdropimg')
+                        // Drop event doesn't work in case that ExtendedTextInteraction is included
+                        // so for the prevent dropping - just make element as disabled, the restore its state
+                        .on(namespaceHelper.namespaceAll('dragover', 'preventdropimg'), function (event) {
+                            // if img then I don't want to do drop
+                            if (dropped.tagName === 'IMG'
+                                || dropped.tagName === 'OBJECT'
```

Comment:
Please move all the restricted tags name in the config, so we can adapt them if needed : 
```
if( dropped && (_.contains(restrictedTags, dropped.tagName.toLowerCase()) ) {
```